templates/deployment-zabbix-server.yaml

```diff
@@ -109,7 +117,11 @@ spec:
 {{- toYaml .Values.zabbixagent.resources | nindent 12 }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
+          {{- if .Values.zabbixagent_image.tag }}
```

Comment:
Hi @scoopex!

Can you change

{{- if .Values.zabbixagent_image.tag }}

by

{{- if .Values.zabbixagent.image.tag }}

?

Reply:
Sure, done :-)